src/reason-parser/reason_lexer.mll

```diff
@@ -608,7 +587,7 @@ rule token = parse
   *)
   | "}"  { RBRACE }
   | ">}" { GREATERRBRACE }
-  | "=<" is_jsx_not_infix_operator  {
+  | "=<" uppercase_or_lowercase+ {
```

Comment:
Cool, we might run into a few more cases like this and if we do, the fix could be similar to this one.